addons/l10n_es_edi_tbai/models/l10n_es_edi_tbai_certificate.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from base64 import b64decode
+from OpenSSL import crypto
```

Comment:
see how [this comment](https://github.com/odoo/odoo/pull/83883/files#diff-8902cfd630f1d73c831fb20d5567b9b2f2566e9357296a44ec12b23ff8c3ab07R33-R35) removes that requirement
```suggestion
```